### PR TITLE
Add support for blockfrost api

### PR DESCRIPTION
**Pull request by Sital999**

### CODE REVIEW

File: agent-manager/.env.example [hunk 1]
```diff
@@ -7,6 +7,10 @@ CARDANO_NETWORK_MAGIC=4
 
 KUBER_BASE_URL=
 KUBER_API_KEY=
+BLOCKFROST_API_KEY=
+BLOCKFROST_BASE_URL=
```
Comment (mesudip) [inline agent-manager/.env.example:11]: Lets use the NETWORK= preview|preprod|sanchonet|mainnet to directly determine BLOCKFROST_BASE_URL.

File: agent-manager/.env.example [hunk 2]
```diff
@@ -7,6 +7,10 @@ CARDANO_NETWORK_MAGIC=4
...
+BLOCKFROST_SUBMIT_API=
```
Comment (mesudip) [inline agent-manager/.env.example:12]: rename the env variable

File: agent-manager/.env.example [hunk 3]
```diff
@@ -20,3 +24,6 @@ METADATA_BASE_URL=
 
 DB_SYNC_BASE_URL=
 
+PORT=
```
Comment (mesudip) [inline agent-manager/.env.example:27]: rename SERVER_PORT

Comment (mesudip) [reply agent-manager/.env.example:27]: also fill in the defaults.

File: agent-manager/src/service/TxBuildAndSubmit/TxSubmitterService.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            return new Blockfrost()
+    }
+}
+
+export class TxSubmitterService {
+
+    submitter: KuberType | BlockFrostType
+
+    constructor(txBuilderType: TxSubmitterType) {
+        this.submitter = fetchTxBuilder(txBuilderType)
```
Comment (mesudip) [inline agent-manager/src/service/TxBuildAndSubmit/TxSubmitterService.ts:24]: reaname it to getTxSubmitter

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
```
Comment (mesudip) [inline pyproject.toml:1]: remove file

**Review by mesudip — CHANGES_REQUESTED** [review]

File: agent-manager/.env.example (hunk 1 above)
Comment (Sital999) [reply agent-manager/.env.example:11]: Done

File: agent-manager/.env.example (hunk 3 above)
Comment (Sital999) [reply agent-manager/.env.example:27]: Done

File: pyproject.toml (hunk 5 above)
Comment (Sital999) [reply pyproject.toml:1]: Done

File: agent-manager/src/service/TxBuildAndSubmit/TxSubmitterService.ts (hunk 4 above)
Comment (Sital999) [reply agent-manager/src/service/TxBuildAndSubmit/TxSubmitterService.ts:24]: Done

**Review by mesudip — APPROVED** [review]

**Pull request merged**